test/child/ChildERC20.test.js

```diff
@@ -62,18 +66,22 @@ contract('ChildERC20', async function(accounts) {
 
     // withdraw those tokens
     const receipt = await childToken.withdraw(amount)
-
-    receipt.logs.should.have.lengthOf(2)
-
-    receipt.logs[0].event.should.equal('Withdraw')
-    receipt.logs[0].args.token.should.equal(rootToken.address)
-    receipt.logs[0].args.user.should.equal(owner)
-    receipt.logs[0].args.amount.toString().should.equal(amount)
-
-    receipt.logs[1].event.should.equal('LogWithdraw')
-    receipt.logs[1].args.input1.should.be.bignumber.equal(amount)
-    receipt.logs[1].args.amount.should.be.bignumber.equal(amount)
-    receipt.logs[1].args.output1.should.be.bignumber.equal(0)
+    receipt.logs.should.have.lengthOf(3)
+
+    receipt.logs[0].event.should.equal('Transfer')
+    receipt.logs[0].args.from.toLowerCase().should.equal(owner)
+    receipt.logs[0].args.to.toLowerCase().should.equal(ZeroAddress)
+    receipt.logs[0].args.value.should.be.bignumber.equal(amount)
+
+    receipt.logs[1].event.should.equal('Withdraw')
```

Comment:
Remove `Withdraw` and use `Transfer` only.

Reply:
we are not getting that token address in transfer event so, will keep withdraw event for now and optimize on sending params part and just send only withdraw event logs.